### PR TITLE
Fix realm explorer crash

### LINKED ISSUE #4092: Realm Explorer crashes in a release build
## Describe the bug

In a realease build (8.1.0-rc0) typign anything into the search bar of the realm explorer crashes the app

### To reproduce

1. Go to Settings
2. Click on the search bar and try to search for something

### Expected behaviour

No crash

### Proposed Solution

N/A

### Version and device info

- App version: 8.1.0-rc0
- Tablet model:
- OS version:

### Additional context

N/A

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #4092 #3048 

## Change summary

Always thought it odd that the table search wasn't debounced.
Added some debounce, and prevented searching for tables with name < 3 chars
Also caught and toasted the error, if there is one.

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Explore. With Realm

### Related areas to think about

Now looking at #3048 to actually show data

### CODE REVIEW

**Comment by mark-prins** [comment]
Bonus! fixed #3048. It's a little iffy but is working :)

File: src/pages/RealmExplorer.js [hunk 1]
```diff
@@ -73,8 +74,8 @@ const getRealmObjectsFields = ({ schema: objectSchemas }) =>
     })
     .reduce((acc, object) => ({ ...acc, ...object }));
 
-const REALM_OBJECTS = getRealmObjects(schema);
 const REALM_OBJECTS_FIELDS = getRealmObjectsFields(schema);
+const REALM_OBJECTS = Object.keys(REALM_OBJECTS_FIELDS);
```
Comment (mark-prins) [inline src/pages/RealmExplorer.js:78]: The `REALM_OBJECTS_FIELDS` has correct key names, which relate to table names.
The previous `REALM_OBJECTS` is giving minified versions of the properties, which is causing havoc.

**Review by joshxg — APPROVED** [review]
Yup it's great.. Thanks!!

**Comment by ArpitaSussol** [comment]
Tested in V8.1.0 rc1

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [x] Explore. With Realm

**Pull request merged**